src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -76,53 +76,50 @@ function toggleChevronImage(panelCollapse, toggleChevron) {
  * Expands panel, showing the spreadsheet interface.
  */
 function expandStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('show');
-    $(panelCollapse[0]).addClass('checked');
+    panelCollapse.collapse('show');
 }
 
 /**
  * Collapses panel, hiding the spreadsheet interface.
  */
 function collapseStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('hide');
-    $(panelCollapse[0]).removeClass('checked');
+    panelCollapse.collapse('hide');
 }
 
 /**
  * Expands/Collapses the panel depending on the current state of the panel.
  */
 function toggleStudentsPanel($panelHeading, panelCollapse, displayIcon, toggleChevron) {
+    let isPanelExpanded = false;
     displayIcon.html('');
-    if ($(panelCollapse[0]).hasClass('checked')) { // panel is shown
+    if (panelCollapse.hasClass('in')) { // panel is shown
         collapseStudentsPanel(panelCollapse);
     } else {
         expandStudentsPanel(panelCollapse);
+        isPanelExpanded = true;
     }
-    toggleChevronImage(panelCollapse, toggleChevron);
+    toggleChevronImage(isPanelExpanded, toggleChevron);
+    return isPanelExpanded;
```

Comment:
This is not a good practice. The function is trying to do too many things, both computing something and applying some visual changes. From the function's name, it should only apply some visual changes. Leave the computing something to some other functions, or in this case since the computation is pretty simple, just recompute it where this function is called.